presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -109,6 +113,26 @@ Operations
 
     Returns the geometry value that represents the point set symmetric difference of two geometries.
 
+.. function:: ST_Buffer(Geometry, double) -> Geometry
+
+    Returns the geometry value that represents all points whose distance from any point of an input geometry is less than or equal to a specified distance
+
+.. function:: point_to_bing_tile(Geometry, int) => varchar
```

Comment:
Parameters in documentation are the logical names, not the type. So name it `zoom` or `zoom_level` rather than `int`